lib/definition_table_generator.rb

```diff
@@ -9,6 +9,10 @@ def terminology_table(content_key, version)
   DefinitionTableGenerator.new(version, nil).terminology_table(content_key)
 end
 
+def patch_definition_table(content_key, action, version)
```

Comment:
You could probably get by with just content_key and version as parameters since the action will only ever be patch

Reply:
True. Fixed. https://github.com/cerner/fhir.cerner.com/commit/b3be566e03125de4bdc242726080efeaaef55e65#diff-6590262e197180cc09f6ad70a2a01a5bR35